http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -319,6 +411,12 @@ class GKEInterpreter[F[_]: Parallel: ContextShift](
                                        KubernetesNamespace(getAppResult.app.appResources.namespace.name))
       _ <- appQuery.markAsDeleted(getAppResult.app.id, now).transaction
     } yield ()
+
+  private def retry[A](ioa: F[A], retryable: Throwable => Boolean): F[A] =
```

Comment:
This method also exists in `DataprocInterpreter` (https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/DataprocInterpreter.scala#L617) was wondering where is a good place to pull this method out to so it can be used both here and in `DataprocInterpreter`

Reply:
Yeah I wonder if we could call the wb-libs `retry`, which I did in `VPCInterpreter`: https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/VPCInterpreter.scala#L150

Reply:
I can try that out!